src/gen9_avc_encoder.c

```diff
@@ -74,6 +75,14 @@
         OUT_BCS_BATCH(batch, attr);                             \
     } while (0)
 
+/* FEI specific buffer sizes per MB in bytes for gen9 */
+#define FEI_AVC_MB_CODE_BUFFER_SIZE      64
+#define FEI_AVC_MV_DATA_BUFFER_SIZE      128
+#define FEI_AVC_MB_CONTROL_BUFFER_SIZE   16
+#define FEI_AVC_MV_PREDICTOR_BUFFER_SIZE 40
+#define FEI_AVC_DISTORTION_BUFFER_SIZE   48
```

Comment:
if these defines are specified for Gen9 and different from Gen95 or later, can we add the prefix or postfix to identify them clearly